bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/internal/handler/OpenWebNetAuxiliaryHandler.java

```diff
@@ -52,10 +51,22 @@ public class OpenWebNetAuxiliaryHandler extends OpenWebNetThingHandler {
     public static final Set<ThingTypeUID> SUPPORTED_THING_TYPES = OpenWebNetBindingConstants.AUX_SUPPORTED_THING_TYPES;
     private final Logger logger = LoggerFactory.getLogger(OpenWebNetAuxiliaryHandler.class);
 
+    private static final Set<OpenWebNetAuxiliaryHandler> auxHandlers = new HashSet<>();
```

Comment:
Except addind to and removing from this set, it looks like it is not used ?
And why static ?